File: notifications/rest/services/list-service/list-service.rb
# NOTE: This example uses the next generation Twilio helper library - for more
# information on how to download and install this version, visit
# https://www.twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = '{{ auth_token }}'
client = Twilio::REST::Client.new account_sid, auth_token

services = client.notifications.v1.services.list(limit: 100)

puts services

File: notifications/rest/services/list-service/list-service.java
// NOTE: This example uses the next generation Twilio helper library - for more
// information on how to download and install this version, visit
// https://www.twilio.com/docs/libraries/java
import com.twilio.sdk.Twilio;
import com.twilio.sdk.resource.ResourceSet;
import com.twilio.sdk.resource.notifications.v1.Service;

import java.util.Iterator;

public class ListService {
  // Find your Account Sid and Token at twilio.com/user/account
  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
  public static final String AUTH_TOKEN = "{{ auth_token }}";

  public static void main(String args[]) {
    // Initialize the client
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    // List the services
    ResourceSet<Service> services = Service.read().execute();
    Iterator<Service> iterator = services.iterator();
    while (iterator.hasNext()) {
      Service service = iterator.next();
      System.out.println(service.toString());
    }
  }
}